tests/test_notebooks.py

```diff
@@ -269,13 +271,13 @@ def dynamic_analysis(self, path, kernel_name):
 
         # Execute the notebook and allow errors (run all cells), output is
         # written to a temporary file which is automatically deleted.
-        with tempfile.NamedTemporaryFile(suffix='.ipynb') as fout:
+        with tempfile.NamedTemporaryFile(suffix='.ipynb',mode='wb+',delete=False) as fout:
             args = ['jupyter', 'nbconvert', 
```

Comment:
What was the motivation for this change? Mode is same as default and we do want the output file to automatically be deleted after it is analyzed.

Reply:
I'll remove the mode here.  I had different modes explicitly stated while I was troubleshooting an issue here.  

The delete=False is here to address an issue in Windows that seems to have with temporary files (see https://bugs.python.org/issue14243).  Would just deleting the file after we're done with everything be sufficient?

Reply:
Got it. Please leave the `delete=False`, and modify the comment above it to explain that the setting is due to issues with windows, possibly include the link to the bug. 

Hopefully it will be fixed in the near future and we can revert to the original settings. Given that this is an eight year old issue I don't expect much, but then again it recently received some attention. Hope springs eternal.